js/flightlog_fields_presenter.js

```diff
@@ -502,16 +512,26 @@ function FlightLogFieldPresenter() {
                 return (value + 1500).toFixed(0) + " us";
             case 'rcCommand[3]':
                 return value.toFixed(0) + " us";
-                
+
             case 'motor[0]':
-            case 'motor[1]':            
-            case 'motor[2]':            
-            case 'motor[3]':            
-            case 'motor[4]':            
-            case 'motor[5]':            
-            case 'motor[6]':            
-            case 'motor[7]':            
-                return Math.round(flightLog.rcMotorRawToPct(value)) + " %";
+            case 'motor[1]':
+            case 'motor[2]':
+            case 'motor[3]':
+            case 'motor[4]':
+            case 'motor[5]':
+            case 'motor[6]':
+            case 'motor[7]':
+                return `${flightLog.rcMotorRawToPctEffective(value).toFixed(1)} %`;
+
+            case 'motorRaw[0]':
+            case 'motorRaw[1]':
+            case 'motorRaw[2]':
+            case 'motorRaw[3]':
+            case 'motorRaw[4]':
+            case 'motorRaw[5]':
+            case 'motorRaw[6]':
+            case 'motorRaw[7]':
+                return `${flightLog.rcMotorRawToPctPhysical(value).toFixed(1)} % (${value})`;
```

Comment:
And `% absolute` here.

Reply:
I tried to restrict the size to minimum. We have not too much space here, but I will try.